Jellyfin.Api/Models/PlaylistDtos/CreatePlaylistDto.cs

```diff
@@ -31,4 +32,14 @@ public class CreatePlaylistDto
     /// Gets or sets the media type.
     /// </summary>
     public MediaType? MediaType { get; set; }
+
+    /// <summary>
+    /// Gets or sets the playlist users.
+    /// </summary>
+    public IReadOnlyList<PlaylistUserPermissions> Users { get; set; } = [];
+
+    /// <summary>
+    /// Gets or sets a value indicating whether the playlist is public.
+    /// </summary>
+    public bool Public { get; set; } = true;
```

Comment:
I should've caught this one earlier, but we might want to rename this to "IsPublic because most of our other models do the same (look at BaseItemDto for example, with isLive/isMovie/isFolder etc.).

Personally I'd prefer just "Public" but consistency might be the better way to go.

Reply:
(same for the UpdatePlaylistDto model)